mkdocs.yml

```diff
@@ -99,6 +93,13 @@ nav:
   - 'Wallets and Seeds': 'faq/wallets-and-seeds.md'
   - 'Blocks': 'faq/blocks.md'
   - 'Common Errors and Solutions': 'faq/errors.md'
+  - 'PoW': 'faq/proof-of-work-mining.md'
```

Comment:
This needs to be removed from its original position - it still exists under the "Proof-of-Work Mining" top level header.

Reply:
Oops. Fixed